docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,30 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test alternative deployments and options.
+
+There are various alternate configurations you can test with local-up-cluster.  These can be useful when prototyping a new feature.
+
+#### Authentication via Kubeconfig files
+
+You can test that kubeconfig files and certificates are working properly using local-up-cluster.
+
+An example kubeconfig file can be found at [Kubeconfig file](../user-guide/kubeconfig-file.md), consult the docs for other details on authentication/authorization.
+
+The key thing to remember is that the apiserver launches with certain credentials that kubelets or other users will need to provide when connecting, unless you use the insecure endpoint.
+
+#### NodePorts or ClusterIPs as service endpoint for kubelets to access load balanced API Servers
+
+First, you MUST launch BOTH the apiserver and kubelet with corresponding authentication credentials (the simplest way is to use a token file with a kubeconfig file passed to the kubelet).
+
+Then, you can use ClusterIP or NodePort to provide a loadbalanced endpoint for the API servers to your kubelet, like so:
+
+- ClusterIP: Launch kubelets with the `--api-servers="https://10.0.0.1:443"` option.
+
+- NodePort: Modify the apiserver launch to include a NodePort range i.e. `--service-node-port-range="30000-40000"` with `--apiserver-node-port="30279"`, and launch the kubelets with an `--api-servers="https://127.0.0.1:30279"` option.
```

Comment:
Okay if I simply change 127.0.0.1 to "x.y.z.a" ?  The combination of command line options is IMO has pedagogical value, right?